### PR TITLE
Update guidance pages

### PR DESCRIPTION
**Pull request by karlchillmaid**
This pull request adds and updates guidance content for:

* guidance (landing page)
* email branding
* letter branding
* receive text messages
* reply-to email address
* send files by email
* reusable templates
* text message sender
* upload a letter

It also updates the navigation so these pages will appear in the left-hand navigation.

This is the first iteration of a series of changes to guidance and features content and navigation.

### CODE REVIEW

File: app/templates/views/guidance/index.html [hunk 1]
```diff
@@ -20,8 +20,12 @@ <h1 class="heading-large">Guidance</h1>
 
   <ul class="govuk-list govuk-list--bullet">
     <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.edit_and_format_messages') }}">edit and format messages</a></li>
-    <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.branding_and_customisation') }}">add branding and customisation</a></li>
+    <li>change your <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.email_branding') }}">email branding</a></li>
+    <li>add <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.letter_branding') }}">letter branding</a></li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.receive_text_messages') }}">receive text messages</a></li>
+    <li>add a <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.reply_to_email_address') }}">reply-to email address</a></li>
     <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.send_files_by_email') }}">send files by email</a></li>
+    <li>send messages using <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.templates') }}">templates</a></li>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/index.html:28]:
```suggestion
    <li>use message <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.templates') }}">templates</a></li>
```

File: app/templates/views/guidance/text-message-sender.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  <p class="govuk-body">To change the text message sender from the default of ‘GOVUK’:</p>
+
+  <ol class="govuk-list govuk-list--number">
+    <li>Go to the <b class="govuk-!-font-weight-bold">Text message settings</b> section of the {{ service_link(current_service, 'main.service_settings', 'settings') }} page.</li>
+    <li>Select <b class="govuk-!-font-weight-bold">Manage</b> on the <b class="govuk-!-font-weight-bold">Text message senders</b> row.</li>
+    <li>Select <b class="govuk-!-font-weight-bold">Change</b> or <b class="govuk-!-font-weight-bold">Add text message sender</b>.</li>
+</ol>
+
+<p class="govuk-body">If you want to receive text messages, your sender name will be an 11-digit phone number.</p>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/text-message-sender.html:26]:
```suggestion
<p class="govuk-body">If you want to <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.receive_text_messages') }}">receive text messages</a>, your sender name will be an 11-digit phone number.</p>
```

File: app/templates/views/guidance/upload-a-letter.html [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
   <h1 class="heading-large">Upload a letter</h1>
 
-  <p class="govuk-body">Upload a one-off letter as a PDF and we’ll print, pack and post it for you.</p>
+  <p class="govuk-body">Upload a one-off letter as a PDF and Notify will print, pack and post it for you.</p>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/upload-a-letter.html:16]:
```suggestion
  <p class="govuk-body">Upload a PDF of a single letter and Notify will print, pack and post it for you.</p>
```

File: app/templates/views/guidance/upload-a-letter.html [hunk 4]
```diff
@@ -13,7 +13,7 @@
 
   <h1 class="heading-large">Upload a letter</h1>
 
-  <p class="govuk-body">Upload a one-off letter as a PDF and we’ll print, pack and post it for you.</p>
+  <p class="govuk-body">Upload a PDF of a single letter and Notify will print, pack and post it for you.</p>
   <p class="govuk-body">You can use this feature if you send a lot of one-off letters or if our reusable letter templates do not meet your needs.</p>
 
   <p class="govuk-body">To upload and send a letter from a PDF file:</p>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/upload-a-letter.html:19]:
```suggestion
  <p class="govuk-body">To upload and send a letter:</p>
```

File: app/main/views/index.py [hunk 5]
```diff
@@ -244,26 +244,41 @@ def guidance_index():
...
-        "views/guidance/create-and-send-messages.html",
+        "views/guidance/letter-branding.html",
         navigation_links=using_notify_nav(),
     )
 
 
-@main.route("/using-notify/guidance/edit-and-format-messages")
-def edit_and_format_messages():
+@main.route("/using-notify/guidance/receive-text-messages")
+def receive_text_messages ():
```
Comment (karlchillmaid) [inline app/main/views/index.py:271]:
```suggestion
def receive_text_messages():
```

File: app/main/views/index.py [hunk 6]
```diff
@@ -244,26 +244,41 @@ def guidance_index():
...
+def receive_text_messages():
     return render_template(
-        "views/guidance/edit-and-format-messages.html",
+        "views/guidance/receive-text-messages.html",
+        navigation_links=using_notify_nav(),
+    )
+
+
+@main.route("/using-notify/guidance/reply-to-email-address")
+def reply_to_email_address ():
```
Comment (karlchillmaid) [inline app/main/views/index.py:279]:
```suggestion
def reply_to_email_address():
```

File: app/main/views/index.py [hunk 7]
```diff
@@ -244,26 +244,41 @@ def guidance_index():
...
+@main.route("/using-notify/guidance/edit-and-format-messages")
+def edit_and_format_messages():
     return render_template(
-        "views/guidance/branding-and-customisation.html",
+        "views/guidance/edit-and-format-messages.html",
         navigation_links=using_notify_nav(),
     )
 
+@main.route("/using-notify/guidance/email-branding")
+def email_branding():
```
Comment (karlchillmaid) [inline app/main/views/index.py:255]:
```suggestion
def guidance_email_branding():
```

File: app/main/views/index.py [hunk 8]
```diff
@@ -244,26 +244,41 @@ def guidance_index():
...
+    return render_template(
+        "views/guidance/email-branding.html",
+        navigation_links=using_notify_nav(),
+    )
 
-@main.route("/using-notify/guidance/create-and-send-messages")
-def create_and_send_messages():
+
+@main.route("/using-notify/guidance/letter-branding")
+def letter_branding():
```
Comment (karlchillmaid) [inline app/main/views/index.py:263]:
```suggestion
def guidance_letter_branding():
```

File: app/main/views/index.py [hunk 9]
```diff
@@ -244,26 +244,41 @@ def guidance_index():
...
-        "views/guidance/create-and-send-messages.html",
+        "views/guidance/letter-branding.html",
         navigation_links=using_notify_nav(),
     )
 
 
-@main.route("/using-notify/guidance/edit-and-format-messages")
-def edit_and_format_messages():
+@main.route("/using-notify/guidance/receive-text-messages")
+def receive_text_messages():
```
Comment (karlchillmaid) [inline app/main/views/index.py:271]:
```suggestion
def guidance_receive_text_messages():
```

File: app/main/views/index.py [hunk 10]
```diff
@@ -276,6 +291,22 @@ def send_files_by_email():
     )
 
 
+@main.route("/using-notify/guidance/templates")
+def templates():
```
Comment (karlchillmaid) [inline app/main/views/index.py:295]:
```suggestion
def guidance_templates():
```

File: app/main/views/index.py [hunk 11]
```diff
@@ -276,6 +291,22 @@ def send_files_by_email():
...
+    return render_template(
+        "views/guidance/templates.html",
+        navigation_links=using_notify_nav(),
+    )
+
+
+@main.route("/using-notify/guidance/text-message-sender")
+def text_message_sender():
```
Comment (karlchillmaid) [inline app/main/views/index.py:303]:
```suggestion
def guidance_text_message_sender():
```

File: app/main/views/index.py [hunk 12]
```diff
@@ -292,6 +323,14 @@ def letter_specification():
     )
 
 
+@main.route("/using-notify/guidance/create-and-send-messages")
+def create_and_send_messages():
```
Comment (karlchillmaid) [inline app/main/views/index.py:327]:
```suggestion
def guidance_create_and_send_messages():
```

File: app/main/views/sub_navigation_dictionaries.py [hunk 13]
```diff
@@ -77,13 +77,29 @@ def using_notify_nav():
...
                 },
                 {
-                    "name": "Branding",
-                    "link": "main.branding_and_customisation",
+                    "name": "Email branding",
+                    "link": "main.email_branding",
+                },
+                {
+                    "name": "Letter branding",
+                    "link": "main.letter_branding",
```
Comment (tombye) [inline app/main/views/sub_navigation_dictionaries.py:85]:
```suggestion
                    "link": "main.guidance_letter_branding",
```

File: app/main/views/sub_navigation_dictionaries.py [hunk 14]
```diff
@@ -77,13 +77,29 @@ def using_notify_nav():
...
+                },
+                {
+                    "name": "Receive text messages",
+                    "link": "main.receive_text_messages",
```
Comment (tombye) [inline app/main/views/sub_navigation_dictionaries.py:89]:
```suggestion
                    "link": "main.guidance_receive_text_messages",
```

File: app/main/views/sub_navigation_dictionaries.py [hunk 15]
```diff
@@ -77,13 +77,29 @@ def using_notify_nav():
...
+                },
+                {
+                    "name": "Reply-to email address",
+                    "link": "main.reply_to_email_address",
```
Comment (tombye) [inline app/main/views/sub_navigation_dictionaries.py:93]:
```suggestion
                    "link": "main.guidance_reply_to_email_address",
```

File: app/main/views/sub_navigation_dictionaries.py [hunk 16]
```diff
@@ -77,13 +77,29 @@ def using_notify_nav():
                     "link": "main.edit_and_format_messages",
                 },
                 {
-                    "name": "Branding",
-                    "link": "main.branding_and_customisation",
+                    "name": "Email branding",
+                    "link": "main.email_branding",
```
Comment (tombye) [inline app/main/views/sub_navigation_dictionaries.py:81]:
```suggestion
                    "link": "main.guidance_email_branding",
```

File: app/templates/views/guidance/index.html [hunk 17]
```diff
@@ -20,8 +20,12 @@ <h1 class="heading-large">Guidance</h1>
 
   <ul class="govuk-list govuk-list--bullet">
     <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.edit_and_format_messages') }}">edit and format messages</a></li>
-    <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.branding_and_customisation') }}">add branding and customisation</a></li>
+    <li>change your <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.email_branding') }}">email branding</a></li>
```
Comment (tombye) [inline app/templates/views/guidance/index.html:23]:
```suggestion
    <li>change your <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_email_branding') }}">email branding</a></li>
```

File: app/templates/views/guidance/index.html [hunk 18]
```diff
@@ -20,8 +20,12 @@ <h1 class="heading-large">Guidance</h1>
...
+    <li>add <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.letter_branding') }}">letter branding</a></li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.receive_text_messages') }}">receive text messages</a></li>
+    <li>add a <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.reply_to_email_address') }}">reply-to email address</a></li>
```
Comment (tombye) [inline app/templates/views/guidance/index.html:26]:
```suggestion
    <li>add a <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_reply_to_email_address') }}">reply-to email address</a></li>
```

File: app/templates/views/guidance/schedule-messages.html [hunk 19]
```diff
@@ -12,15 +12,16 @@
...
     <li>Go to the {{ service_link(current_service, 'main.choose_template', 'templates') }}.</li>
-    <li>Add a new template or choose an existing template and select <b class="govuk-!-font-weight-bold">Get ready to send</b>.</li>
+    <li>Add a new template or choose an existing template.</li>
+    <li>Select <b class="govuk-!-font-weight-bold">Get ready to send</b>.</li>
     <li>Upload a list or recipients.</li>
     <li>Set the date and time you want to send your messages.</li>
   </ol>
 
-  <p class="govuk-body">Your scheduled messages will appear on the dashboard and uploads pages.</p>
+  <p class="govuk-body">Your scheduled messages will appear on the {{ service_link(current_service, 'main.dashboard', 'dashboard') }} and {{ service_link(current_service, 'main.uploads', 'uploads') }} pages.</p>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/schedule-messages.html:26]: @samuelhwilliams I think adding these link broke the checks. I can’t find the correct routes for these, though. Can you help?

Comment (samuelhwilliams) [reply app/templates/views/guidance/schedule-messages.html:26]:
```suggestion
  <p class="govuk-body">Your scheduled messages will appear on the {{ service_link(current_service, 'main.service_dashboard', 'dashboard') }} and {{ service_link(current_service, 'main.uploads', 'uploads') }} pages.</p>
```

File: app/main/views/index.py [hunk 20]
```diff
@@ -244,58 +244,90 @@ def guidance_index():
     )
 
 
-@main.route("/using-notify/guidance/branding-and-customisation")
-def branding_and_customisation():
+@main.route("/using-notify/guidance/edit-and-format-messages")
```
Comment (samuelhwilliams) [inline app/main/views/index.py:247]: We need to add a redirect for this - which new page should anyone who hits this end up at (eg if they've bookedmarked it)?

File: app/main/views/index.py [hunk 21]
```diff
@@ -244,58 +244,90 @@ def guidance_index():
     )
 
 
-@main.route("/using-notify/guidance/branding-and-customisation")
```
Comment (samuelhwilliams) [inline app/main/views/index.py:247]: We need to add a redirect for this - which new page should anyone who hits this end up at (eg if they've bookedmarked it)?

File: app/main/views/index.py [hunk 22]
```diff
@@ -244,58 +244,90 @@ def guidance_index():
...
+@main.route("/using-notify/guidance/edit-and-format-messages")
+def guidance_edit_and_format_messages():
     return render_template(
-        "views/guidance/branding-and-customisation.html",
+        "views/guidance/edit-and-format-messages.html",
         navigation_links=using_notify_nav(),
     )
 
 
-@main.route("/using-notify/guidance/create-and-send-messages")
```
Comment (samuelhwilliams) [inline app/main/views/index.py:255]: Do we maybe need a redirect for this as well? I can't immediately tell - I can access it on the live site but I don't think it's linked to from anywhere yet.

File: app/main/views/index.py (hunk 21 above)
Comment (karlchillmaid) [reply app/main/views/index.py:247]: There isn’t an obvious answer, as the content from that page has been split into 4 new pages.

Best bet is to redirect them to `using-notify/guidance`, so they can navigate to the content they’re interested in.

File: app/main/views/index.py [hunk 23]
```diff
@@ -317,6 +350,7 @@ def old_page_redirects():
         "main.old_using_notify": "main.using_notify",
         "main.information_risk_management": "main.security",
         "main.old_integration_testing": "main.integration_testing",
+        "main.old_schedule_messages": "main.guidance_schedule_messages",
```
Comment (samuelhwilliams) [inline app/main/views/index.py:353]:
```suggestion
        "main.old_schedule_messages": "main.guidance_schedule_messages",
        "main.old_branding_and_customisation": "main.guidance_index",
```

File: app/main/views/index.py [hunk 24]
```diff
@@ -307,6 +339,7 @@ def schedule_emails_and_text_messages():
 @main.route("/terms", endpoint="old_terms")
 @main.route("/information-security", endpoint="information_security")
 @main.route("/using_notify", endpoint="old_using_notify")
+@main.route("/using-notify/guidance/schedule-emails-and-text-messages", endpoint="old_schedule_messages")
```
Comment (samuelhwilliams) [inline app/main/views/index.py:342]:
```suggestion
@main.route("/using-notify/guidance/schedule-emails-and-text-messages", endpoint="old_schedule_messages")
@main.route("/using-notify/guidance/branding-and-customisation", endpoint="old_branding_and_customisation")
```

File: app/main/views/index.py (hunk 22 above)
Comment (karlchillmaid) [reply app/main/views/index.py:255]: It was never linked to or signposted.

As a precaution we could redirect to `using-notify/guidance` here as well.

File: tests/app/test_navigation.py [hunk 25]
```diff
@@ -189,6 +196,7 @@
             "old_guest_list",
```
Comment (samuelhwilliams) [inline tests/app/test_navigation.py:196]:
```suggestion
            "old_branding_and_customisation",
            "old_guest_list",
```

File: app/main/views/index.py (hunk 22 above)
Comment (samuelhwilliams) [reply app/main/views/index.py:255]: I think we can leave it out of redirects if that's the case 👍

**Review by samuelhwilliams — APPROVED** [review]
This looks good to me now :) 🎉  Nice work everyone

**Pull request merged**